src/geoadmin.rs

```diff
@@ -15,14 +15,17 @@
 //! let res = geoadmin.forward(&address);
 //! assert_eq!(res.unwrap(), vec![Point::new(7.451352119445801, 46.92793655395508)]);
 //! ```
+use std::fmt::Debug;
```

Comment:
Unused now? (and in all the other files)

Reply:
You are right! Strangely, Clippy did not complain to me..